DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompleteByDate.cshtml

```diff
@@ -25,9 +25,9 @@
 
 @section NavBreadcrumbs {
   @if (Model.AccessedVia.Equals(DelegateProgressAccessRoute.CourseDelegates)) {
-    <partial name="~/Views/TrackingSystem/Delegates/Shared/_CourseDelegatesBreadcrumbs.cshtml" model="Model.CustomisationId" />
+    <partial name="~/Views/TrackingSystem/Delegates/Shared/_CourseDelegatesProgressBreadcrumbs.cshtml" model="(Model.CustomisationId, Model.ProgressId)" />
   } else {
-    <partial name="~/Views/TrackingSystem/Delegates/Shared/_ViewDelegateBreadcrumbs.cshtml" model="Model.DelegateId" />
+    <partial name="~/Views/TrackingSystem/Delegates/Shared/_ViewDelegateProgressBreadcrumbs.cshtml" model="(Model.CustomisationId, Model.ProgressId)" />

```

Comment:
The first parameter of the `_ViewDelegateProgressBreadcrumbs` model should be the DelegateID instead of the customisationID. Looks like customisationID has accidentally been used here and on the other two files.